omc3/scripts/lhc_corrector_list_check.py

```diff
@@ -0,0 +1,92 @@
+""" 
+Corrector Lists Check
+---------------------
+
+Quick check of the corrector lists against the LHC sequence.
+It checks if all the correctors in the LHC sequence are used in the variables.
+
+These checks are performed against the correction knobs used for global corrections.
+"""
+from pathlib import Path
+import re
+from omc3.model.accelerators.lhc import Lhc
+from omc3.model.constants import AFS_ACCELERATOR_MODEL_REPOSITORY
+
+ACC_MODELS_LHC: Path = AFS_ACCELERATOR_MODEL_REPOSITORY / "lhc" 
+LHC_SEQ_FILE: str = "lhc.seq"
+
+
+def parse_lhc_sequence(year: str) -> dict[int, set[str]]:
+    """ Find all the correctors in the LHC sequence.
+    They follow the pattern 'kq[^fsx][.a-z0-9]+' 
```

Comment:
I see some `kq10` in the json files of this PR.

Does the last part of this regex match from 0 to 9 while allowing repeats of the matched elements (in which case, match 1 then match 0)? RegExr did not tell me

Reply:
`+` matches 1 or more